### PR TITLE
Manually update buildah to v1.13.1

### PR DESCRIPTION
**Pull request by mtrmac**
Should help with #791.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,24 +3,53 @@ module github.com/containers/skopeo
...
 
 require (
-	github.com/containers/buildah v1.12.0 // indirect
-	github.com/containers/common v0.0.7
-	github.com/containers/image/v5 v5.0.1-0.20191126085826-502848a1358b
-	github.com/containers/ocicrypt v0.0.0-20190930154801-b87a4a69c741
-	github.com/containers/storage v1.15.3
+	github.com/containerd/cgroups v0.0.0-20200116170754-a8908713319d // indirect
+	github.com/containerd/containerd v1.3.2 // indirect
+	github.com/containerd/continuity v0.0.0-20200107194136-26c1120b8d41 // indirect
```
Comment (TomSweeneyRedHat) [inline go.mod]: I'm a little surprised to see the containerd additions here?  They're not in Buildah....

Comment (mtrmac) [reply go.mod]: Ah, right, `go get -u` means “for all dependencies of $target, updates them”. I’ll rework this. Thanks!

Comment (mtrmac) [reply go.mod]: Updated now.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**